chia/wallet/did_wallet/did_wallet.py

```diff
@@ -526,7 +540,8 @@ async def create_attestment(
 
         message_spend_bundle = SpendBundle([message_spend], AugSchemeMPL.aggregate([]))
         # sign for AGG_SIG_ME
-        message = innermessage + coin.name() + self.wallet_state_manager.constants.AGG_SIG_ME_ADDITIONAL_DATA
+        to_sign = Program.to([coin.puzzle_hash, coin.amount, innermessage]).get_tree_hash()
```

Comment:
Question about our new definition of coins in tx outputs (now using parent_id):

Might that change warrant changing some internals of the did wallet?